### DISCUSSION TITLE
source socket for syslog

### DISCUSSION THREAD

**Discussion by coredump17** [vectordotdev/vector#19303] [General]
Hi All,

I need to receive syslog messages that do not confirm to any the standards (yay!). I used UDP socket and send onto kafka for further processing via another vector process. I have noticed that i cannot see the message 'Facility' or 'Severity' when using socket. Do i have any options to pull this out of the packet when converting to json?

Do we have a function to convert the priority to facility and severity?

thanks in advance!

**Comment by jszwedko** [comment]
🤔 You could do manual parsing in a `remap` transform using regular expressions.

**Reply by coredump17** [reply]
Thanks, done this. I just though a function would be useful in future to help calculate this

**Comment by coredump17** [comment]
Used remap to calculate facility and severity
